app/views/green_lanes/category_assessments/_category.html.erb

```diff
@@ -0,0 +1,32 @@
+<% if @goods_nomenclature.applicable_category_assessments.any? %>
```

Comment:
I think this needs an `else` - rather than showing a empty section, say that there are no applicable category assessements

Reply:
Done